tests/tests/common/docker.rs

```diff
@@ -187,8 +187,18 @@ impl DockerTestClient {
         Ok(mapped_ports)
     }
 
-    /// halts execution until a trigger message is detected on stdout
-    pub async fn wait_for_message(&self, trigger_message: &[u8]) -> Result<&Self, DockerError> {
+    /// halts execution until a trigger message is detected on stdout or, optionally,
+    /// waits for a specified amount of time, disregarding the message.
```

Comment:
I was thinking this wait would happen after waiting for the message.

Reply:
Oh, sure, I'll fix that